paperqa/settings.py

```diff
@@ -678,6 +678,19 @@ def _deprecated_field(self) -> Self:
                 setattr(self.agent.index, new_name, value)  # Propagate to new location
         return self
 
+    @model_validator(mode="after")
+    def _validate_temperature_for_o1_preview(self) -> Self:
+        """Ensures temperature is 1 if the LLM is 'o1-preview' or 'o1-mini'.
+
+        o1 reasoning models only support temperature = 1.  See
+        https://platform.openai.com/docs/guides/reasoning/quickstart
+        """
+        if self.llm.startswith("o1-") and self.temperature != 1:
+            raise ValueError(
+                f"the temperature must be set to 1, {self.temperature} was specified."
```

Comment:
```suggestion
                "When dealing with OpenAI o1 models,  the temperature"
                f" must be set to 1, {self.temperature} was specified."
```

Sorry to double post here, but do you mind specifying "o1" somewhere in this message? It will make the failure message more useful